apps/nextjs-website/next.config.js

```diff
@@ -1,4 +1,7 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {}
+const nextConfig = {
+    output: 'export',
+    assetPrefix: process.env.CDN_PATH,
```

Comment:
Removing `assetPrefix` creates a broken build when ran locally, so it was added to at least give the option to quickly add the `CDN_PATH` environment variable to be able to make sure everything renders/loads correctly (instead of having to wait for a developer to push the update if need be).

I would personally want to deploy a test website before entirely removing the value.

We could simply build two test websites (one with `assetPrefix` and one without) and run it in an environment resembling our production environment.

Reply:
Keeping `assetPrefix` also has the slight advantage of making testing builds locally that much smoother, while `CDN_PATH` should be able to just be left empty in production environments without repercussions.

Reply:
I didn't know it would break; actually, I haven't tried to build it 😄